.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+<!--
+
+    Copyright (c) 2019-present Sonatype, Inc.
+    This program and the accompanying materials are made available under
+    the terms of the Eclipse Public License 2.0 which accompanies this
+    distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+
+-->
+
+# How to be a contributor to this project
+
+## Talk to a human first
+
+Looking to dive in? FANTASTIC! Reach out to one of our experts and we can help. Hit us up on our Slack channel #react-components.
```

Comment:
Should be create a "For Sonatypers" section and move the mentions of slack and JIRA to it?

Reply:
https://github.com/sonatype/sonatype-react-shared-components/pull/46/commits/58ad6690752611df2ed4799d4b5271a8f8e205c6